File: src/android/flc-exoplayer/src/main/java/pl/fulllegitcode/exoplayer/Log.java
package pl.fulllegitcode.exoplayer;

public class Log {

  private static final String TAG = "FlcExoPlayer";

  public static void error(String message) {
    android.util.Log.e(TAG, message);
  }

  public static void info(String message) {
    android.util.Log.i(TAG, message);
  }

}

File: src/unity/Assets/FlcExoPlayer/Scripts/Log.cs
using UnityEngine;

namespace FullLegitCode.ExoPlayer
{
    internal class Log
    {
        const string TAG = "[FlcExoPlayer]";

        public static void Error(string message)
        {
            Debug.LogError(TAG + " " + message);
        }

        public static void Info(string message)
        {
            Debug.Log(TAG + " " + message);
        }
    }
}
